src/components/input/input.scss

```diff
@@ -91,6 +91,12 @@ md-input-container {
     bottom: 100%;
     @include rtl(left, 0, auto);
     @include rtl(right, auto, 0);
+
+    &.md-required:after {
+      content: ' *';
```

Comment:
@DevVersion I know this is old, but I thought I would update this since I just looked at it. The labels are properly "aligned" to the right, it's just that the width is not 100% of the parent. I guess we need to use `right: 0` in addition to `text-align: right` here to fix?